extensions/cli/geoserver/src/main/java/mil/nga/giat/geowave/cli/geoserver/GeoServerPasswordConverter.java

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
this class seems unnecessary (and is not currently used).  I think we can safely get rid of it.